diff --git a/linux/rk3588-mnt-pocket-reform.dts b/linux/rk3588-mnt-pocket-reform.dts
index a79a8d6..fdbfdc0 100644
--- a/linux/rk3588-mnt-pocket-reform.dts
+++ b/linux/rk3588-mnt-pocket-reform.dts
@@ -346,8 +346,8 @@
 	tlv320aic3100: codec@18 {
 		compatible = "ti,tlv320aic3100";
 		reg = <0x18>;
-		assigned-clocks = <&cru I2S0_8CH_MCLKOUT>;
-		assigned-clock-rates = <12288000>;
+		// assigned-clocks = <&cru I2S0_8CH_MCLKOUT>;
+		// assigned-clock-rates = <12288000>;
 		clocks = <&cru I2S0_8CH_MCLKOUT>;
 		clock-names = "mclk";
 		ai31xx-micbias-vg = <2>;
